Cargo.toml

```diff
@@ -1,15 +1,16 @@
 [package]
 name = "Amethyst"
-version = "3.6.0"
-authors = ["michal <michal@tar.black>", "axtlos <axtlos@tar.black>"]
+version = "4.0.0"
+authors = ["michal <michal@tar.black>", "axtlos <axtlos@tar.black>", "trivernis <trivernis@protonmail.com>"]
 edition = "2021"
 description = "A fast and efficient AUR helper"
-repository = "https://github.com/crystal-linux/amethyst"
 license-file = "LICENSE.md"
-keywords = ["aur", "crystal-linux", "pacman", "aur-helper"]
-categories = ["command-line-utilities"]
 default-run = "ame"
 
+[features]
+pkg-warner = []
+
```

Comment:
Some of the Cargo.toml file seems to have bit the dust, we no longer have the pkg-warner feature and i'd like to keep the `repository`, `keywords`, and `categories` keywords, so we can push to crates.io at some point